daemons/controld/controld_schedulerd.c

```diff
@@ -209,15 +217,16 @@ do_pe_control(long long action,
               enum crmd_fsa_input current_input, fsa_data_t * msg_data)
 {
     if (action & A_PE_STOP) {
-        pe_subsystem_free();
+        controld_clear_fsa_input_flags(R_PE_REQUIRED);
+        handle_disconnect();
```

Comment:
We have to actually disconnect here, before handle_disconnect()

Reply:
Oh, of course.  Fixed.